config/routes.rb

```diff
@@ -90,10 +87,10 @@
   resources :speakers, only: [:destroy]
   resources :events, only: [:index]
 
-  resources :event_teammate_invitations, only: :show, param: :slug do
+  resources :teammates, only: :show, param: :token do
```

Comment:
Using resources here seems like overkill.  Also, are we even using show now?  If not, I would just make two  routes instead.